src/libs/io/adapter/dbus/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+include (LibAddMacros)
+
+if (NOT ENABLE_IO_ADAPTER_DBUS)
+	# required for debian-musl build - not other purpose
```

Comment:
"not other purpose"?

And as written above: Please directly check for the reason why the compilation fails.

Reply:
Should have been "no other purpose" :)